examples/OWNERS

```diff
@@ -5,8 +5,10 @@ reviewers:
   - gabemontero
   - soltysh
   - danmcp
+  - adambkaplan
 approvers:
   - bparees
   - smarterclayton
   - csrwng
   - pmorie
+  - adambkaplan
```

Comment:
i believe being in a toplevel file carries down to any subdirs, so you probably didn't need to add yourself to the subdirs explicitly.